### ISSUE TITLE
Feature modification: holding click on number should increase

### ISSUE THREAD

**Issue by binx33** [SabakiHQ/Sabaki#749]
Based on this clip from popular streamer: https://clips.twitch.tv/RamshackleColdJackalDansGame

Basically when you click&drag while using the number tool, it copies the number. It does seem more useful to make it increase in my opinion.

**Comment by yishn** [comment]
dwyrin was playing on OGS in the linked clip, not on Sabaki. Currently, Sabaki does not allow drag & drop markers like that at all.

**Comment by binx33** [comment]
Awww. Sorry for the waste of time, my mistake.

That being said it could be a nice addition to be able to maintain click to duplicate shapes and do that thing with numbers, but I guess you can just close this topic if it's complicated and not useful enough.

**Comment by yishn** [comment]
No, it's a nice feature request, thanks for bringing this up to my attention.
